src/common/FastRational.h

```diff
@@ -640,32 +691,50 @@ inline void addition(FastRational& dst, const FastRational& a, const FastRationa
 inline void substraction(FastRational& dst, const FastRational& a, const FastRational& b) {
     if (a.wordPartValid() && b.wordPartValid()) {
         if (b.num == 0) {
-            dst.num = a.num;
-            dst.den = a.den;
-        }
-        else if (a.num == 0) {
-            if (b.num == WORD_MIN)
-                goto overflow;
-            dst.num = -b.num; // This overflows if b = -2147483648
+            dst = a;
```

Comment:
Use the old implementation `dst.num = a.num; dst.den = a.den;`

Reply:
What about this one?

Reply:
Well that's an open-ended question :)

How about:

```suggestion
    if (b.wordPartValid() && b.num == 0) {
            dst = a;
            return;
        } else if (a.wordPartValid() && b.wordPartValid() {
```

Reply:
Let me commit that so it gets more clear.

Reply:
In this suggestion, you would duplicate the check.
I like the original version.
```suggestion
            dst.num = a.num;
            dst.den = a.den;
```

Reply:
Ok, the new commit is hopefully in line with what you suggest here.